cli/src/main/kotlin/OrtMain.kt

```diff
@@ -80,11 +82,19 @@ data class GlobalOptions(
  */
 fun main(args: Array<String>) {
     Os.fixupUserHomeProperty()
-    OrtMain().main(args)
+
+    try {
+        OrtMain().main(args)
+    } finally {
+        OrtMain.logger.debug { "ORT command execution terminated." }
+    }
+
     exitProcess(0)
 }
 
 class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
+    companion object : Logging
```

Comment:
This is not needed thanks to some extension-property magic specific to `CliktCommand`.

Reply:
Ah, but then I need an instance of `OrtMain` right? An expression like `OrtMain.logger` does not work.

Reply:
See https://github.com/oss-review-toolkit/ort/pull/5690#discussion_r954597355.